cmd/prometheus/main.go

```diff
@@ -564,6 +569,12 @@ func main() {
 		ruleManager *rules.Manager
 	)
 
+	if cfg.enableAutoGOMAXPROCS {
+		if _, err := maxprocs.Set(); err != nil {
+			level.Warn(logger).Log("auto_gomaxprocs", "failed", "msg", "Failed to set GOMAXPROCS automatically", "err", err)
```

Comment:
```suggestion
			level.Warn(logger).Log("msg", "Failed to set GOMAXPROCS automatically", "err", err)
```

to match what we have everywhere else in the codebase.

Reply:
Ok, sorry.

I've merged suggestions from you and Peter.